bedrock/settings/__init__.py

```diff
@@ -18,9 +18,7 @@
     try:
         from .local import *  # noqa
     except ImportError as exc:
-        exc.args = tuple(['%s (did you rename bedrock/settings/local.py-dist?)' %
-                          exc.args[0]])
-        raise exc
+        'local.py is supported, but no longer necessary'
```

Comment:
Yes. Good call. That's what I was thinking but commented in the wrong place. Could print a deprecation warning in an `else` block here.